airbyte-integrations/connectors/source-facebook-marketing/unit_tests/test_base_insight_streams.py

```diff
@@ -327,7 +327,7 @@ def test_incremental_lookback_period_updated(self, api, monkeypatch, set_today):
         start_date = pendulum.parse("2020-03-01")
         end_date = pendulum.parse("2020-05-01")
         yesterday, _ = set_today("2020-04-01")
-        monkeypatch.setattr(AdsInsights, "INSIGHTS_LOOKBACK_PERIOD", pendulum.duration(days=20))
+        monkeypatch.setattr(AdsInsights, "insights_lookback_period", pendulum.duration(days=20))
```

Comment:
Can we remove this `monkeypatch` and pass `insights_lookback_window` as parameter to `AdsInsights` constructor like this:

```
stream = AdsInsights(api=api, start_date=start_date, end_date=end_date, insights_lookback_window=20)
```

?